src/engine/menus.cpp

```diff
@@ -850,7 +863,10 @@ static struct applymenu : menu
         g.start(menu_start, NULL, true);
         g.text("the following settings have changed:");
         g.pushfont("little");
-        loopv(needsapply) g.text(needsapply[i].desc, 0xFFFFFF, "point");
+        for (int i = 0; i < needsapply.length(); i++)
```

Comment:
I would suggest not to just stubbornly replace all occurrences of the macro, but just go for the more modern iterator-based loops where applicable. How about

```suggestion
        for (const auto& i : needsapply)
```

If you can *not* use this approach, please don't use the deprecated `length()` but the STL-compatible `size()`.

*Applies to pretty much all loops you modified.*

Reply:
Yeah, I'll do that